Cargo.toml

```diff
@@ -95,10 +95,11 @@ version = "1"
 version = "0.27"
 
 [dependencies.serde]
-version = "1"
+version = "1.0.130"
```

Comment:
these version changes don't do anything (which is confirmed by `Cargo.lock` not changing); in other words, snarkOS already uses these versions

Reply:
Ah good catch. It was mainly just the use of the precision feature that was needed for `serde_json`